base-ci-resource.yaml

```diff
@@ -91,11 +93,13 @@ testNodeApplicationUsingPostgres:
   script:
     - cp -r $CI_PROJECT_DIR/sql/* /flyway/sql/
     - sh /flyway/flyway -url=jdbc:postgresql://$POSTGRES_SERVER:5432/$POSTGRES_DB -schemas=public -user=$POSTGRES_USER -password=$POSTGRES_PASSWORD -placeholders.ENVIRONMENT=dev migrate
+    - $TYPESCRIPT && npm run build
```

Comment:
NB: 

Could even take it a step further with logic like 

`cat package.json  | jq '.scripts.build'; if (($? == 0)); then npm run build fi;`

Wouldn't even need to configure it. The script would just run a build step if there is a build step.

Reply:
i think it just needs a semicolon after build? i think i've forgotten everything i knew about bash

Reply:
bash syntax is also particularly obnoxious.

Reply:
Sorry, this comment I made was probably a *bad* idea...

Brandon had a good way of making these comments by giggling and saying 5 times "This is a really bad idea. Like, you should never do this, but you could ...." Then leaving it up to me if I wanted to use his bad idea.

Maybe I should self-identify bad ideas better. Sorry if you put a lot of time into this and didn't like the result.

Reply:
I took a break from this, jq isn't installed. I'm sure I could force it if I wanted to, but, I'll just go back to the variable.